sources/packages/web/src/views/institution/locations/program-info-request/LocationEditProgramInfoRequest.vue

```diff
@@ -176,18 +176,22 @@ export default {
           },
         });
       } catch (error) {
-        const errorLabel = "Unexpected error!";
+        let errorLabel = "Unexpected error!";
         let errorMsg =
           "An error happened while saving the Program Information Request.";
-        [
+        const isCustomError = [
           OFFERING_START_DATE_ERROR,
           INVALID_STUDY_DATES,
           OFFERING_INTENSITY_MISMATCH,
-        ].forEach((customError) => {
-          if (error.includes(customError)) {
-            errorMsg = error.replace(customError, "").trim();
-          }
-        });
+          APPLICATION_DATE_OVERLAP_ERROR,
+        ].some(
+          (errorCode: string) => errorCode === error.response.data?.errorType,
+        );
+
+        if (isCustomError) {
+          errorLabel = "Application Overlap";
```

Comment:
I forgot to mention this point when we spoke. The error label must be Generic to all these custom errors. 
OFFERING_START_DATE_ERROR,
          INVALID_STUDY_DATES,
          OFFERING_INTENSITY_MISMATCH,
          APPLICATION_DATE_OVERLAP_ERROR,